File: examples/petstore/oseg_generated/php/LogoutUser_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

try {
    (new OpenAPI\Client\Api\UserApi($config))->logoutUser();
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling User#logoutUser: '
        . print_r($e->getResponseObject());
}

File: examples/petstore/oseg_generated/csharp/LogoutUser_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class LogoutUserDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            new UserApi(config).LogoutUser();
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling User#LogoutUser: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
